src/homePage/homePage.component.tsx

```diff
@@ -218,29 +178,38 @@ const HomePage = (): React.ReactElement => {
           </Box>
         </div>
       </div>
-      <Box className={classes.contentBox}>
-        <Paper className={classes.paper} elevation={1}>
+      <Box
+        sx={{
+          transform: 'translate(0px, -20px)',
+          marginLeft: '8%',
+          marginRight: '8%',
+        }}
+      >
+        <Paper sx={paperStyles} elevation={1}>
           <Grid container style={{ height: '100%' }}>
             <Grid item xs={6}>
-              <Box className={classes.paperContent}>
-                <Typography variant="h3" className={classes.paperMainHeading}>
-                  {t('home-page.browse.title')}
-                </Typography>
+              <Box sx={paperContentStyles}>
                 <Typography
-                  variant="body1"
-                  className={classes.paperDescription}
+                  variant="h3"
+                  sx={(theme) => ({
+                    fontWeight: 'bold',
+                    fontSize: '32px',
+                    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Can remove this ignore, you can also remove the cast to any and the optional chaining operators

Reply:
Yes, this was because it was copied across from datagateway, but I can remove it.